tox.ini

```diff
@@ -2,25 +2,24 @@
 skipsdist = True
 usedevelop = True
 envlist =
-    py{36,37,38}-dj{22,30,31}-wt211
```

Comment:
I have configured tox to test earlier django versions (>=3.0) was wagtail 2.15 can support those versions